kip-0017.md

```diff
@@ -111,6 +111,9 @@ This contains information on which of the requested items it supports.
 }
 ```
 
+There are also `optionalNamespaces` that **may** be included, but are not required in the settlement from the wallet.  
+The structure for these is the same as the structure for the `requiredNamespaces`.
```

Comment:
Added to extra context and links relevant to the difference between `optionalNamespaces` and `requiredNamespaces`.

```suggestion
There are also [`optionalNamespaces`](https://docs.walletconnect.com/2.0/specs/clients/sign/namespaces#proposal-namespace) that **may** be included, but are not required in the settlement from the wallet.  
The structure for these is the same as the structure for the `requiredNamespaces`.

The dApp uses the `requiredNamespaces` and `optionalNamespaces` fields to indicate to the wallet what chains, events, and methods it would like permission to.

The main difference being that the wallet must grant access to all the chains, methods, and events specified in the `requiredNamespaces` in order to establish an active session.

But it is up to the wallet to satisfy all, none, or some of the chains, methods, and events specified in the `optionalNamespaces` field.
```

Reply:
Very well stated.